stingray/fourier.py

```diff
@@ -2294,13 +2309,14 @@ def avg_pds_from_events(
     mean : float
         the mean flux
     """
-    if segment_size is None:
-        segment_size = gti.max() - gti.min()
-    n_bin = int(segment_size / dt)
-    if fluxes is None:
-        dt = segment_size / n_bin
+    binned = fluxes is not None
```

Comment:
I am confused by this: if this function generates an averaged PDS from events, then why is there even a question whether it's binned or not? I'm assuming the answer has to be no?

Reply:
The function is called from events, but it does work with any time series. I will take the chance to rename it

Reply:
Renamed and deprecated the old name!